query/reverse.js

```diff
@@ -75,6 +75,18 @@ function generateQuery( clean ){
     logStr += '[param:boundary_circle] ';
   }
 
+  // for coarse reverse when boundary circle radius is undefined
+  if( check.number(clean['boundary.circle.lat']) &&
```

Comment:
Any reason not to combine this block with the one above it? Checking for lat/lon and then inside the if checking radius and setting the value accordingly?